amm/drink-tests/src/stable_swap_tests/mod.rs

```diff
@@ -12,7 +12,7 @@ use drink::{self, runtime::MinimalRuntime, session::Session, AccountId32};
 use ink_primitives::AccountId;
 use ink_wrapper_types::{Connection, ToAccountId};
 
-pub const FEE_BPS_DENOM: u128 = 10_000;
+pub const FEE_DENOM: u128 = 1_000_000_000;
```

Comment:
you have this in `constants` already. Can you import it?

Reply:
I wanted to avoid importing `amm_helpers` just for a few constants but if that makes more sense we can do that.

Reply:
Hmm, OK. Just add a comment here.